core/templates/pages/story-editor-page/navbar/story-editor-navbar.directive.html

```diff
@@ -46,7 +46,7 @@
     <button type="button" ng-click="unpublishStory()"
             ng-if="isStoryPublished()"
             class="btn btn-secondary protractor-test-unpublish-story-button"
-            ng-disabled="isStorySaveable()">
+            ng-disabled="isStorySaveable() || getWarningsCount() > 0 || validationIssues.length > 0 || explorationValidationIssues.length > 0 || prepublishValidationIssues.length > 0">
```

Comment:
Unpublish story need not be disabled with these cases, right?
It should be possible to unpublish a story if there are no pending changes to be saved.

Reply:
Removed the extra checks. PTAL!